### PR TITLE
Feature/tdr 1/phpunit8 upgrade

### PR DESCRIPTION
**Pull request by Yury-Razhkou**
related to: https://oat-sa.atlassian.net/browse/TDR-1
depends on: https://github.com/oat-sa/generis/pull/751
Upgrade to phpunit8

### CODE REVIEW

File: test/unit/model/storage/SqlTest.php [hunk 1]
```diff
@@ -56,22 +53,22 @@ public function testCastValue($fieldName, $value, $expectedType)
...
-            [Sql::DIAGNOSTIC_OSVERSION, '8', 'string'],
-            [Sql::DIAGNOSTIC_BROWSERVERSION, 33.02, 'string'],
-            [Sql::DIAGNOSTIC_BROWSERVERSION, 73, 'string'],
-            ['another', '33.02', 'float'],
-            ['another3', 'omg', 'string'],
-            ['another2', 33.02, 'float'],
+            ['size', 1, 'assertIsInt'],
+            ['size', 't1', 'assertIsString'],
+            [Sql::DIAGNOSTIC_BANDWIDTH_SIZE, '12', 'assertIsInt'],
+            [Sql::DIAGNOSTIC_INTENSIVE_BANDWIDTH_SIZE, '12.00', 'assertIsInt'],
```
Comment (mike-ionut-mihai-sandu-tao) [inline test/unit/model/storage/SqlTest.php]: this is a little bit weird

Comment (Yury-Razhkou) [reply test/unit/model/storage/SqlTest.php]: Yes. i'm also don't like this solution. But i have no other ideas to quick fix this.

File: test/unit/model/authorization/AnonymousTest.php [hunk 2]
```diff
@@ -48,7 +49,7 @@ public function testIsAuthorized()
 
     public function testGetAuthorizationUrl()
     {
-        $this->setExpectedException('oat\taoClientDiagnostic\exception\InvalidCallException');
+        $this->expectException(InvalidCallException::class);
         $this->instance->getAuthorizationUrl('urlFixture');
     }
 }
```
Comment (bartlomiejmarszal) [inline test/unit/model/authorization/AnonymousTest.php]:
```suggestion
}

```

File: test/unit/model/storage/SqlTest.php (hunk 1 above)
Comment (bartlomiejmarszal) [reply test/unit/model/storage/SqlTest.php]: I agree. It's not but this fix keep consistent with other changes so I'm willing to accept it.

**Pull request merged**